source/blog/2016-07-25-ember-2-7-and-2-8-beta-released.md

```diff
@@ -0,0 +1,200 @@
+  a route map.
+
+By making these APIs public, ember-engines and other addons will be able to
+access them without a feature-flag in a (soon-to-be) stable release of Ember. By
+accessing only public interfaces in Ember, an addon won't need to rely on
+private overrides and can provide as stable an experience as possible.
+
+We predict that engines will grow in popularity once work on lazy loading is
+complete. By delaying the loading of engines until they're accessed, we can
+decrease the initial payload size and startup time for applications. If you're
+interested in helping to implement or test lazy loading of engines, please check
+out the thorough [attack plan](https://github.com/dgeb/ember-engines/issues/154)
+written up by [@nathanhammond](https://github.com/nathanhammond).
+
+If you'd like to get started using engines, check out the excellent
+[guides](https://github.com/dgeb/ember-engines/blob/master/guides/01-introduction.md)
+written by [@trentmwillis](https://github.com/trentmwillis).
+
+#### `Enumerable#includes` and `Array#includes`
+
+In an effort to remain inline with ES standards, the methods
+`Enumerable#contains` and `Array#contains` have been deprecated in favor of the
+new methods `Enumerable#includes` and `Array#includes`.
+
+Thanks to [@alexspeller](https://github.com/alexspeller) for the [RFC](https://github.com/emberjs/rfcs/blob/master/text/0136-contains-to-includes.md)
+that proposed this change.
+
+Thanks as well to [@bmeurant](https://github.com/bmeurant) for the
+[PR](https://github.com/emberjs/ember.js/pull/13553) that implemented the change.
+
+#### `Ember.String.isHTMLSafe`
+
+The new method `Ember.String.isHTMLSafe` detects if a string was decorated using
+`Ember.String.htmlSafe`.
+
+For example:
+
+```javascript
+var plainString = 'plain string',
+    safeString = Ember.String.htmlSafe('<div>someValue</div>');
+
+Ember.String.isHTMLSafe(plainString); // false
+Ember.String.isHTMLSafe(safeString);  // true
+```
+
+Thanks to [@workmanw](https://github.com/workmanw) for the
+[RFC](https://github.com/emberjs/rfcs/pull/139) that proposed this new API, as
+well as the [PR](https://github.com/emberjs/ember.js/pull/13666) that
+implemented it.
+
+On a related note, please create safe strings with `Ember.String.htmlSafe`
+instead of the deprecated `Ember.Handlebars.SafeString`. See the [deprecation
+guide](http://emberjs.com/deprecations/v2.x/#toc_use-ember-string-htmlsafe-over-ember-handlebars-safestring)
+for details.
+
+#### `Ember.Test.checkWaiters`
+
+The new method `Ember.Test.checkWaiters` provides a simple mechanism for test
+tooling to determine whether all async test waiters have settled. This replaces
+the intimate API `Ember.Test.waiters`, which has been deprecated and will be
+removed in Ember 2.8 final.
+
+Thanks to [@rwjblue](https://github.com/rwjblue) and
+[@krisselden](https://github.com/krisselden) for implementing this method.
+
+----
```

Comment:
Probably can mention that 2.8 stable will the LTS candidate and link to http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html